DQMServices/Components/plugins/DQMStoreStats.cc

```diff
@@ -62,25 +62,25 @@ class DQMStoreStatsSubfolder {
     ++totalHistos_;
     totalBins_ += nBins;
     totalEmptyBins_ += nEmptyBins;
-    totalMemory_ += (nBins *= sizeof(float));
```

Comment:
nBins is passed by value, makes no sense to modify it

Reply:
indeed